components/automate-cli/cmd/chef-automate/backup.go

```diff
@@ -785,17 +797,43 @@ func runDeleteBackupCmd(cmd *cobra.Command, args []string) error {
 			"Must specify a backup",
 		)
 	}
+	var location string
+	start := 0
+	var newArgs []string
+	var backup []*api.BackupTask
+
+	if strings.Contains(args[0], "/") || strings.Contains(args[0], "\\") {
+		location = args[0]
+		start = 1
+	}
+
+	backup, err := getBackupTask(location)
+	if err != nil {
+		return errors.Errorf("Unable to fetch the backup list with error: %v", err.Error())
+	}
+
+	backupMap := getBackupMapFromBackupList(backup)
 
 	ids, err := idsToBackupTasks(args)
 	if err != nil {
 		return err
+	}
 
+	for i := start; i < len(args); i++ {
```

Comment:
We need to test this logic when we have flags passed as well.

Reply:
according to this document https://docs.chef.io/automate/backup/, if we see the example we can't pass any flag.